updatecli/updatecli.d/updatecanal.yaml

```diff
@@ -0,0 +1,87 @@
+---
+name: "Update Canal version" 
+
+sources:
+  flannel:
+    name: Get flannel version
+    kind: githubrelease
+    spec:
+      owner: rancher
+      repository: image-build-flannel
+      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "UPDATECLI_GITHUB_ACTOR" }}'
+      typefilter:
+        latest: true
+        release: true
+        draft: false
+        prerelease: false
+      versionfilter:
+        kind: regex
+        # pattern accepts any semver constraint
+        pattern: "v[0-9]+.[0-9]+.[0-9]+-build[0-9]+"
+  calico:
+    name: Get calico version
+    kind: githubrelease
+    spec:
+      owner: rancher
+      repository: image-build-calico
+      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "UPDATECLI_GITHUB_ACTOR" }}'
+      typefilter:
+        latest: true
+        release: true
+        draft: false
+        prerelease: false
+      versionfilter:
+        kind: regex
+        # pattern accepts any semver constraint
+        pattern: "v[0-9]+.[0-9]+.[0-9]+-build[0-9]+"
+
+targets:
+  flannelImage:
+    name: "Bump to latest flannel version on the chart"
+    kind: shell
+    scmid: default
+    sourceid: flannel
+    spec:
+      command: 'updatecli/scripts/update-canal.sh'
+      environments:
+        - name: FLANNEL_VERSION
+          value: '{{ source "flannel" }}'
+  calicoImage:
+    name: "Bump to latest calico version on the chart"
+    kind: shell
+    scmid: default
+    sourceid: calico
+    dependson:
+      - flannelImage
+    spec:
+      command: 'updatecli/scripts/update-canal.sh'
+      environments:
+        - name: CALICO_VERSION
+          value: '{{ source "calico" }}'
+
+
+scms:
+  default:
+    kind: github
+    spec:
+      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "UPDATECLI_GITHUB_ACTOR" }}'
```

Comment:
Use this instead: https://github.com/rancher/image-build-cni-plugins/blob/main/updatecli/updatecli.d/updateplugins.yaml#L45-L46, that way we use values.yaml